modules/sticky-table-headers/main/index.coffee

```diff
@@ -43,6 +43,21 @@ cloneEvents = (elem, clone) ->
       cloneChildren = clone.childNodes
       cloneEvents elemChildren[i], cloneChildren[i] for i in [0..elemChildren.length]
 
+getChildrenFromTable = (t, body, single) ->
+  realParents = getChildren(t.select(if body then 'tbody' else 'thead'), 'tr')
+  hx.flatten realParents.map (parent) -> getChildren(hx.select(parent), 'th, td', single)
+
+getChildren = (parent, selector, single) ->
+  children = if single
```

Comment:
Avoided that as `parent.children` is a `NodeList`, not an `Array` and using children would also imply use of `Element.matches` which doesn't work in phantom. 
It's easier to select everything that matches the selector and filter than it is to check if a node matches a selector in this case. 
I'd also assume that filtering on a selection and comparing nodes would be quicker than using `Element.matches` on all the direct children.